### PR TITLE
Document suggested setup of package maintained by group.

### PR DESCRIPTION
**Pull request by voxik**
This is attempt to provide more insight how to setup the package ACL when it should be maintained by FAS group.

### CODE REVIEW

File: doc/groups.rst [hunk 1]
```diff
@@ -36,3 +40,22 @@ which ACL you want to give and on which branch.
...
+* Group should have ``commits`` bit. This allows every member of the
+  group to make changes to package in dist-git. This is actually the main
+  purpose of groups.
+* Add "watchcommits" bit to allow notification of the group members about
+  changes in the package dist-git.
+* Group should have ``watchbugzilla`` permissions to allow group to be
+  notified about bugzilla issues associated with the package.
+* Optionally, the group can become PoC of the package. This effectively
+  results only in change of default assignee in bugzilla, nothing else. Keep
+  in mind, that this does not replace ``watchbugzilla`` bugzilla presmissions,
```
Comment (pypingou) [inline doc/groups.rst]: Typo on the last word here: presmissions :)

File: doc/groups.rst [hunk 2]
```diff
@@ -36,3 +40,22 @@ which ACL you want to give and on which branch.
...
+  since the default assignee is not in CC list of the ticket by default
```
Comment (pypingou) [inline doc/groups.rst]: Do we double-check this? I believe it is true but I might have been wrong

**Comment by voxik** [comment]
1. It seems I was not right about the "default CC", so I dropped the note.
2. I fixed some quotes.
3. I added another bullet suggesting to cleanup the ACLs, since users don't need to be listed anywhere, if they are already member of some group with appropriate rights.

Let me know if I should squash the commits.

**Comment by pypingou** [comment]
Looks all good to me, thanks! :)

**Pull request merged**